user_guide/data_management_workflows.rst

```diff
@@ -0,0 +1,105 @@
+.. _user-guide-workflow-management-basics:
+
+Following a Workflow
+====================
+
+In the process of the work with the customers information saved in the CRM system can be changed: carts turn 
+into orders, what seemed to be a potential opportunity appears not to be worth effort or tuns into a successful deal, 
+customer support requests are processed and closed, and so on.
+These and other processes are reflected in OroCRM with creation of new records, change of record properties and 
+creation of new relations. (For example - a new lead appeared, and while processed, it was related to a customer and a 
+contact, and turned into a new opportunity. 
```

Comment:
OroCRM reflects processes like these by creating new records, changing record properties, and making new relations. For example, a new leads appears and, while it is being processed, it gets related to a customer and a contact, and is turned into a new opportunity.